swagger.yaml

```diff
@@ -4630,3 +4630,35 @@ definitions:
         type: string
         enum: [beta, production]
         description: Level of support of the version.
+
+  operator-property:
+    type: object
+    properties:
+      name:
+        type: string
+        description: Name of the property
+      type:
+        type: string
+        enum: ['boolean', 'string', 'number']
+        description: Type of the property
+      options:
+        type: array
+        items:
+          type: string
+        description: Values to select from
+      description:
+        type: string
+        description: Description of a property
+      default_value:
+        type: string
+        description: Default value for the property
+
+  operator-properties:
+    type: object
+    properties:
+      operator_name:
+        $ref: '#/definitions/operator-type'
```

Comment:
This should be just a `string` since `operator-type` would be used for "builtins" and "OLM"s.